airbyte-cdk/java/airbyte-cdk/acceptance-test-harness/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation libs.bundles.datadog
     implementation group: 'io.swagger', name: 'swagger-annotations', version: '1.6.2'
 
+    implementation project(':airbyte-cdk:java:airbyte-cdk:core')
```

Comment:
This makes me realize that this whole `acceptance-test-harness` package should be a `testFixture` in `core` or elsewhere.